common/src/main/java/com/wynntils/wc/utils/scoreboard/ScoreboardManager.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wc.utils.scoreboard;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.mc.event.ScoreboardSetScoreEvent;
+import com.wynntils.mc.event.WynntilsScoreboardUpdateEvent;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.wc.event.WorldStateEvent;
+import com.wynntils.wc.model.WorldState;
+import com.wynntils.wc.utils.scoreboard.objectives.ObjectiveManager;
+import com.wynntils.wc.utils.scoreboard.quests.QuestManager;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Pattern;
+import net.minecraft.ChatFormatting;
+import net.minecraft.server.ServerScoreboard;
+import net.minecraft.world.scores.Objective;
+import net.minecraft.world.scores.Scoreboard;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class ScoreboardManager {
+
+    private static final String GUILD_ATTACK_UPCOMING = "§b§lUpcoming Attacks:";
+    private static final Pattern GUILD_ATTACK_TERRITORY_PATTERN = Pattern.compile("§b-\\s(\\d+):(\\d+)\\s§3(.+)");
+    private static final Pattern OBJECTIVE_HEADER_PATTERN = Pattern.compile("(★ )?(Daily )?Objectives?:");
+    private static final Pattern GUILD_OBJECTIVE_HEADER_PATTERN = Pattern.compile("(★ )?Guild Obj: (.+)");
+    private static final Pattern PARTY_PATTERN = Pattern.compile("§e§lParty:§6\\s\\[Lv. (\\d+)]");
+    private static final Pattern PARTY_PLAYER_HEALTH_PATTERN =
+            Pattern.compile("-\\s\\[\\|\\|(\\d+)\\|\\|]\\s(.+)\\s\\[(\\d+)]");
+
+    // TimeUnit.Seconds
+    private static final int CHANGE_PROCESS_RATE = 1;
+
+    private static final List<ScoreboardLine> reconstructedScoreboard = new ArrayList<>();
+
+    private static final LinkedList<ScoreboardLineChange> queuedChanges = new LinkedList<>();
```

Comment:
```suggestion
    private static final List<ScoreboardLineChange> queuedChanges = new LinkedList<>();
```

It's nicer to hide implementation type in the declaration. :)

Reply:
Oh, I see, you use it as a queue, so you need the type.